### DISCUSSION TITLE
SmtpSettings in appsettings.json

### DISCUSSION THREAD

**Discussion by Zahar661** [serenity-is/Serenity#6569] [Help] [answered]
Hi all! I can't read the settings from the appsettings.json, am I doing something wrong?
![image](https://user-images.githubusercontent.com/26430136/197418273-67b59942-9dd1-422e-818d-0340957d112b.png)
![image](https://user-images.githubusercontent.com/26430136/197418342-906814e3-e089-4345-bcb6-4a343ae2ca7c.png)
PS: Serenity 5.2.2

**Comment by petebrieso** [comment, marked as answer]
did you configure it in `Startup.cs` ?

eg. 
```cs
public void ConfigureServices(IServiceCollection services) 
{
    ...
    services.Configure<MyApiOptions>(Configuration.GetSection(nameof(MyApiOptions)));
    ...
}
```

**Reply by Zahar661** [reply]
Thank you! exactly what is needed!
![image](https://user-images.githubusercontent.com/26430136/197505890-38d0cf14-6e99-47b2-ba0e-9573460d1666.png)
![image](https://user-images.githubusercontent.com/26430136/197506156-a2dc874f-5627-4f50-a003-5a217a747d06.png)
